config/samples/component-integration-installed/operator_v1beta2_moduletemplate_kcp-module-cred-label.yaml

```diff
@@ -46,7 +46,7 @@ spec:
             - name: oci-registry-cred
               value:
                 operator.kyma-project.io/oci-registry-cred: operator-regcred
-          name: template-operator
+          name: raw-manifest
```

Comment:
you can't directly change the content here, this component descriptor is just a local copy, KLM download the actually content from remote oci registry.

````
ocm get resource europe-west3-docker.pkg.dev/sap-kyma-jellyfish-dev/template-operator//kyma-project.io/template-operator -c 1.6.0 -owide
NAME              VERSION IDENTITY TYPE       RELATION ACCESSTYPE   ACCESSSPEC
config            v1.6.0           yaml       local    localOciBlob {"digest":"sha256:db86408caca4c94250d8291aa79655b84146f9cc45e0da49f05a52b3722d74a0"}
template-operator v1.6.0           helm-chart local    localOciBlob {"digest":"sha256:8f01a545b7f53f91dd9f0bb7b74b30dfa2e571dae8d38fd0f56af66f809378fd"}
````
 
This file should only used for the integration test [here](https://github.com/kyma-project/lifecycle-manager/blob/d51230b9959a0b8a9ae592684e3406f5244909ff/tests/integration/controller/kyma/moduletemplate_test.go#L128).

I assume this outdated file should not affect the change you did, please create a followup issue to update this outdated moduletemplate.

Reply:
this does affect the new changes since we switch on the layer name, and default case is not do anything. And because "template-operator" layer is not called "raw-manifest", it is not rendered by the new changes

Reply:
the local copy is indeed used here: https://github.com/kyma-project/lifecycle-manager/blob/2dcdb77176b7192743822204f50e6cb77b3df96e/tests/integration/controller/kyma/moduletemplate_test.go#L128

Reply:
ok, but then we still need to update remote content.